setup.cfg

```diff
@@ -4,7 +4,7 @@ exclude = */.*/,.*/,__pycache__,node_modules
 # E203 whitespace before ':'
 # E231: missing whitespace after ','
 # E501: line too long
-extend_ignore = E129,E203,E231,E501
+ignore = E101, E111, E114, E115, E116, E117, E121, E122, E123, E124, E125, E126, E127, E128, E129, E131, E133, E2, E3, E5, E501, E701, E702, E703, E704, W1, W2, W3, W503, W504
```

Comment:
What's the difference between `ignore` and `extend_ignore`?

Reply:
I'm curious. Can we leave it as it was? (so it's shorter list)

Reply:
Yes, it must be left as it was
ignore overrides the list of files to be left
extend-ignore extends the list